src/renderer/components/wallet/txs/SendFormBTC.txt

```diff
@@ -0,0 +1,198 @@
+import React, { useCallback, useMemo } from 'react'
+
+import { FeeOptions } from '@thorchain/asgardex-bitcoin/lib/types/client-types'
+import {
+  assetToString,
+  Asset,
+  formatAssetAmountCurrency,
+  assetAmount,
+  formatAssetAmount,
+  bn,
+  baseToAsset,
+  AssetSymbol
+} from '@thorchain/asgardex-util'
+import { Row, Form } from 'antd'
+import BigNumber from 'bignumber.js'
+import * as FP from 'fp-ts/lib/function'
+import * as O from 'fp-ts/lib/Option'
+import { useIntl } from 'react-intl'
+
+import { isBnbAsset, BNB_SYMBOL, BTC_DECIMAL } from '../../../helpers/assetHelper'
+import { sequenceTOption } from '../../../helpers/fpHelpers'
+import { trimZeros } from '../../../helpers/stringHelper'
+import { SendTxParams, AddressValidation } from '../../../services/bitcoin/types'
+import { AssetsWithBalance, AssetWithBalance } from '../../../services/wallet/types'
+import { Input, InputBigNumber } from '../../uielements/input'
+import AccountSelector from '../AccountSelector'
+import * as Styled from './Form.style'
+import { validateTxAmountInput } from './util'
+
+type FormValues = {
+  recipient: string
+  amount: string
+}
+
+type Props = {
+  assetsWB: AssetsWithBalance
+  assetWB: AssetWithBalance
+  onSubmit: ({ to, amount, memo }: SendTxParams) => void
+  changeAssetHandler: (asset: Asset) => void
+  isLoading: boolean
+  addressValidation: AddressValidation
+  feeOptions: O.Option<FeeOptions>
+}
+
+export const SendForm: React.FC<Props> = (props): JSX.Element => {
+  const {
+    onSubmit: onSubmitProp,
+    changeAssetHandler,
+    assetsWB,
+    assetWB,
+    isLoading = false,
+    addressValidation,
+    feeOptions: oFeeOptions
+  } = props
+  const intl = useIntl()
+
+  const [form] = Form.useForm<FormValues>()
+
+  const feeLabel = useMemo(
+    () =>
+      FP.pipe(
+        oFeeOptions,
+        O.fold(
+          () => '--',
+          // TODO (@Veado) Which
+          ({ regular }) => `${AssetSymbol.BTC} ${trimZeros(regular.feeRate.toString())} `
+        )
+      ),
+    [oFeeOptions]
+  )
+
+  const isFeeError = useMemo(() => {
+    return FP.pipe(
+      oFeeOptions,
+      O.fold(
+        // Missing (or loading) fees does not mean we can't sent something. No error then.
+        () => !O.isNone(oFee),
+        ([fee, bnbAmount]) => bnbAmount.amount().isLessThan(fee.amount())
+      )
+    )
+  }, [oFeeOptions])
+
+  const renderFeeError = useMemo(() => {
+    if (!isFeeError) return <></>
+
+    const amount = FP.pipe(
+      oBnbAmount,
+      // no bnb asset == zero amount
+      O.getOrElse(() => assetAmount(0))
```

Comment:
maybe we could use  [ZERO_ASSET_AMOUNT](https://github.com/thorchain/asgardex-electron/blob/master/src/renderer/const.ts#L62) to have a single pointer always ?